### PR TITLE
feat: Add category and rules pages

### PR DESCRIPTION
**Pull request by jalajcodes**
https://deploy-preview-3--max-quiz.netlify.app/pages/category.html
https://deploy-preview-3--max-quiz.netlify.app/pages/rules.html

### CODE REVIEW

File: pages/rules.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            </button>
+          </a>
+        </div>
+      </nav>
+    </header>
+
+    <section class="rules container">
+      <h2>Rules</h2>
+
+      <div class="rules__container">
```
Comment (ArjunJunna) [inline pages/rules.html]: rules__container is not centered.

**Review by ArjunJunna — APPROVED** [review]

File: pages/category.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    <link rel="stylesheet" href="../styles/index.css" />
+    <script src="https://kit.fontawesome.com/8269a6457c.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="nav">
+        <a href="/">
+          <div class="nav__logo">
+            <img src="../images/logo.png" alt="logo" width="200" />
+          </div>
```
Comment (anik31) [inline pages/category.html]: suggestion - it will be good if you put the width in css file

File: pages/category.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+    <section class="quizzes container" id="quizzes">
+      <h2>Movie Quizzes</h2>
+      <div class="grid col-3">
+        <div class="card vertical">
+          <div class="card__image">
+            <img
+              class="img-responsive"
+              src="https://thumbs.dreamstime.com/b/male-hand-holding-megaphone-quiz-time-speech-bubble-loudspeaker-banner-business-marketing-advertising-vector-125104939.jpg"
+              alt=""
```
Comment (anik31) [inline pages/category.html]: suggestion - don't leave alt attribute inside img empty, put some text about the contents of the image

File: styles/index.css [hunk 4]
```diff
@@ -17,10 +17,19 @@ h2 {
   margin: 2rem 0;
```
Comment (anik31) [inline styles/index.css:17]: suggestion - add comments for different sections inside the file, if the code length is more it is easy to find a particular section.

File: pages/rules.html [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Max Quiz</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="https://maxui.netlify.app/components/max.css" />
```
Comment (MayankKumar10) [inline pages/rules.html:14]: Import all css files into root directory and import that root file here you don't have to import each file every time

File: pages/rules.html [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    <link rel="stylesheet" href="../styles/index.css" />
+    <link rel="stylesheet" href="../styles/rules.css" />
+    <script src="https://kit.fontawesome.com/8269a6457c.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="nav">
+        <a href="/">
+          <div class="nav__logo">
+            <img src="../images/logo.png" alt="logo" width="200" />
```
Comment (MayankKumar10) [inline pages/rules.html:24]: Don't use inline styling

File: styles/rules.css [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+.rules__container {
+  box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.1);
+  width: fit-content;
+  padding: 2rem;
+  border: 2px solid var(--primary);
+}
+.rules__container ul {
+  list-style: none;
+  padding: 0;
+}
```
Comment (anik31) [inline styles/rules.css]: suggestion - add some space after each ruleset so that they can be clearly distinguished.

File: styles/index.scss [hunk 8]
```diff
@@ -17,9 +17,19 @@ h2 {
...
 }
 
+.card__image {
+  img:hover {
+    transform: scale(1.1);
+  }
+}
+
 .container {
   max-width: 1280px;
```
Comment (MayankKumar10) [inline styles/index.scss]: Try not to use px instead use rem

**Review by anik31 — COMMENTED** [review]
Everything looks nice, the UI is really cool 💯 I have added some suggestions, please look into it. I f you have time constraints you can work on it later.

File: styles/rules.scss [hunk 9]
```diff
@@ -1,14 +1,16 @@
 .rules {
+  display: grid;
+  place-items: center;
+
   &__container {
-    display: grid;
-    place-items: center;
     box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.1);
```
Comment (MayankKumar10) [inline styles/rules.scss]: Make box-shadow a helper class if you used this property elsewhere

**Review by MayankKumar10 — APPROVED** [review]
Look into these suggestions else everything Looks good to me.

File: styles/index.css (hunk 4 above)
Comment (jalajcodes) [reply styles/index.css:17]: This is automatically generated

File: styles/rules.css (hunk 7 above)
Comment (jalajcodes) [reply styles/rules.css]: automatically generated by sass compiler,

**Pull request merged**